.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -57,8 +57,8 @@ body:
       label: Version
       description: What version of our software are you running?
       options:
-        - 1.3.5 (Default)
-        - older (<1.3.5)
+        - 2.0.0 (Default)
```

Comment:
should this list all 2.x versions or should we simply rename the entry to include all 2.x releases?